drupal/scripts/init_drupal.sh

```diff
@@ -34,7 +34,7 @@ if [[ "${DEV_MODE}" == "true" ]]; then
   drush pm:enable -y devel
 
   # enable dev mode
-  drupal site:mode dev
+  #drupal site:mode dev
```

Comment:
dev mode no longer in use in when DEV_MODE is true?

Reply:
Replaced with new module.